src/libraries/System.Diagnostics.Process/tests/Interop.cs

```diff
@@ -1,145 +1,143 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
 using Microsoft.Win32.SafeHandles;
 using System.ComponentModel;
 using System.Runtime.InteropServices;
 using System.Security.Principal;
 
-namespace System.Diagnostics.Tests
+internal static partial class Interop
 {
-    internal static partial class Interop
```

Comment:
this `Interop` type was causing a conflict with the other `Interop` that we use in product code and because of that I was not able to call `Interop.Kernel32.LoadLibrary` in the test code. I decided to just remove the namespace so it de-facto becames a single `Interop` type.